Makefile

```diff
@@ -117,7 +120,6 @@ cluster-sync-importer: cluster-sync
 cluster-sync-cloner: WHAT = cmd/cdi-cloner
 cluster-sync-cloner: cluster-sync
 
-functest:
-	./hack/build/functests.sh
-
-
+functest-image-host: WHAT=tools/cdi-func-test-file-host-init
```

Comment:
I don't think these are needed anymore. build-cdi-func-test-file-host.sh is called with build, and then both init and http are part of the build target as well.